test/cpp_api_parity/parity_table_parser.py

```diff
@@ -0,0 +1,58 @@
+from collections import namedtuple
+
+ParityStatus = namedtuple('ParityStatus', ['has_impl_parity', 'has_doc_parity'])
+
+'''
+This function expects the parity tracker Markdown file to have the following format:
```

Comment:
It looks like the contents of this file were moved from `test/cpp_api_parity/__init__.py` -- did anything change?

Reply:
Nope nothing changes :)